tests/components/litterrobot/test_switch.py

```diff
@@ -0,0 +1,74 @@
+"""Test the Litter-Robot switch entity."""
+from datetime import timedelta
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.components import litterrobot
+from homeassistant.components.litterrobot.hub import REFRESH_WAIT_TIME
+from homeassistant.components.switch import (
+    DOMAIN as PLATFORM_DOMAIN,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+)
+from homeassistant.const import ATTR_ENTITY_ID, STATE_ON
+from homeassistant.util.dt import utcnow
+
+from .common import CONFIG
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+NIGHT_LIGHT_MODE_ENTITY_ID = "switch.test_night_light_mode"
+PANEL_LOCKOUT_ENTITY_ID = "switch.test_panel_lockout"
+
+
+async def setup_hub(hass, mock_hub):
+    """Load the Litter-Robot switch platform with the provided hub."""
+    hass.config.components.add(litterrobot.DOMAIN)
```

Comment:
We shouldn't add this directly. Setup the integration though the standard interface `await hass.config_entries.async_setup`

https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations